akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -47,7 +60,7 @@ import scala.concurrent.duration._
  */
 class Http2ClientSpec extends AkkaSpecWithMaterializer("""
     akka.http.client.remote-address-header = on
-    akka.http.client.http2.log-frames = on
+    akka.http.client.http2.log-frames = off
```

Comment:
Isn't this good to have for failures, should anyway only be logged when failing since `AkkaSpecWithMaterializer` mixes in `WithLogCapturing`?

Reply:
that's a mistake. I forgot to enable this again.